meson.build

```diff
@@ -390,6 +390,7 @@ executable('fuzz-compose', 'fuzz/compose/target.c', dependencies: test_dep)
 
 # Demo programs.
 executable('rmlvo-to-kccgst', 'test/rmlvo-to-kccgst.c', dependencies: test_dep)
+executable('rmlvo-to-keymap', 'test/rmlvo-to-keymap.c', dependencies: test_dep)
```

Comment:
what I've done in the latest series: removed the RMLVO parsing from `print-compiled-keymap` and the now-obsolete `-k`, so you just run it as `print-compiled-keymap /path/to/keymap`.

I'm not 100% sure of the use of that tool outside the tests though I guess it could be used to verify existing keymaps so I added a commit to allow for absolute paths.